tests/bson/bson-binary-002.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+MongoDB\BSON\Binary::__construct() defaults to TYPE_GENERIC
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
```

Comment:
This include is not necessary and can be removed. It was only included in `bson-binary-001.phpt` for some helper functions used within the test (e.g. `fromPHP()`).

Reply:
Done